serif_shared/src/commonMain/kotlin/xyz/room409/serif/serif_shared/MatrixClient.kt

```diff
@@ -51,19 +51,24 @@ class MatrixLogin(val login_message: String, val mclient: MatrixClient): MatrixS
     }
 }
 class MatrixRooms(val msession: MatrixSession): MatrixState() {
-    fun test(): MatrixState {
-        return MatrixLogin("${msession.test()}, now going back to login for now\n", MatrixClient())
+    fun getRoom(): MatrixState {
+        return MatrixChatRoom(msession)
+    }
+}
+class MatrixChatRoom(val msession: MatrixSession): MatrixState() {
+    fun sendMessage(msg : String): MatrixState {
+        return MatrixLogin("${msession.test(msg)}, now going back to login for now\n", MatrixClient())
```

Comment:
Tiny thing, not sure if it's worth it, but I feel like we might as well stay in the MatrixChatRoom state after sending a message instead of "logging out"

Reply:
I agree, we should probably stay in the same state, but would we handle any sort "logging out" and closing of the client in that case or do we just loop until the user quits with ctrl-c? Do we check for a special input string to quit?

Reply:
I was thinking loop until ctrl-c too! We don't really do any logout that isn't just killing the connection object, so I don't think we lose much by just killing the program.

Reply:
I pushed some changes that quits the loop when you enter in `:q` as the message, at which point it will close the session and go back to login.